Documentation/deploy-master.md

```diff
@@ -60,26 +60,11 @@ Next create a [systemd drop-in][dropins], which is a method for appending or ove
 ExecStartPre=/usr/bin/ln -sf /etc/flannel/options.env /run/flannel/options.env
 ```
 
+[calico-docs]: https://github.com/projectcalico/calico-containers/tree/master/docs/cni/kubernetes
+[network-policy]: https://github.com/caseydavenport/kubernetes/blob/network-policy/docs/admin/network-policy.md#network-policy-in-kubernetes
```

Comment:
How far off is this from being merged? I don't think we should mention this in docs if people can't use it in the targeted version.